docs/model_developers/unmanaged_resources/basic_example.cf

```diff
@@ -0,0 +1,15 @@
+entity InterfaceDiscovery extends std::DiscoveryResource:
+    """
+    A discovery resource that discovers interfaces on a specific host.
+
+    :attr name_filter: If not null, only discover interfaces for which the name matches this regular expression.
+                       Otherwise discover all the interfaces on the host.
+    """
+    string? name_filter = null
+end
+
+InterfaceDiscovery.host [1] -- std::Host
```

Comment:
Shouldn't this be `ip::Host` to make it a valid model? I think we should add a test case that loads these files and verifies that the example code works (as far as possible).